post.md

```diff
@@ -0,0 +1,427 @@
+# Server Actions with User Confirmations in Next.js 13
+
+Next.js 13 has brought exciting new experimental React features to web development. The combination of server side rendered components and server actions have the potential to simplify web development while still giving us the full power of client side React when we need it.
```

Comment:
Later in the article you spell "server side" as "server-side." I recommend making the spelling consistent throughout the article (including the spelling of "client-side").

Reply:
Good callout - found this
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/client-side

recommends `client-side` / `server-side` when used as adjectives for technical documentation

Reply:
Good find!